recipes/_master_slurm_config.rb

```diff
@@ -87,6 +100,20 @@
   mode '0755'
 end
 
+cookbook_file '/home/slurm/slurm_resume' do
```

Comment:
we should move this to a more appropriate directory. Also we might want to use 0700 as mode?